browser-test/src/admin_predicates.test.ts

```diff
@@ -214,6 +214,7 @@ describe('create and edit predicates', () => {
       questionName: 'predicate-currency',
     })
     await adminQuestions.addDateQuestion({questionName: 'predicate-date'})
+    await adminQuestions.addDateQuestion({questionName: 'on-or-after-date-question'})
```

Comment:
Make this and the previous predicate-date name follow a similar naming scheme

Reply:
Still need to do this. Use the predicate-date prefix and update the existing to be specific given the new one is now there too.